### ISSUE TITLE
tensorflow sampled softmax loss

### ISSUE THREAD

**Issue by adrienmialland** [chiphuyen/tf-stanford-tutorials#147]
I am using `tf.sampled_softmax_loss` in a RNN for translation from english to french. This is usefull to speed up the training but, is it possible to get the prediction of the samples used by `tf.sampled_softmax_loss` ?

I'd like to be able to compute the accuracy.

Indeed, what I understand is that `tf.sampled_softmax_loss` acte like a final layer that output all the logits for all the required class, and then apply the sampled algrithm to proccess a "smaller" softmax. But, I can't directly access, at least, the logits.
